catalog/spatial/registry/registry-federation-admin-impl/src/main/java/org/codice/ddf/registry/federationadmin/impl/AdminHelper.java

```diff
@@ -100,4 +116,52 @@ public String getName(Configuration config) {
         return configAdmin.getObjectClassDefinition(config)
                 .getName();
     }
+
+    Map<String, Object> getFilterProperties() throws IOException {
+        Map<String, Object> props = new HashMap<>();
+        Dictionary<String, Object> configProps = configurationAdmin.getConfiguration(
+                REGISTRY_POLICY_PID,
+                null)
+                .getProperties();
+        Map<String, Object> configMap = new HashMap<>();
+        if (configProps == null) {
+            configProps = new Hashtable<>();
+        }
+        Enumeration<String> keys = configProps.keys();
+        while (keys.hasMoreElements()) {
```

Comment:
✏️ This collection creation and iteration exists just to simplify the ability to get default values down on lines 135-138. Alternatively, you could remove the temporary collection and use calls like this:
```java
props.put(FederationAdmin.CLIENT_MODE, Optional.ofNullable(configProps.get(DISABLE_REGISTRY_KEY)).orElse(false));
```

With a language like Kotlin, this could be simplified even further using the Elvis operator to something like
```kotlin
props.put(FederationAdmin.CLIENT_MODE, configProps[DISABLE_REGISTRY_KEY] ?: false)
```